eidpdata/src/main/java/uk/ac/ucl/eidp/data/PoolStrategy.java

```diff
@@ -91,8 +94,12 @@ private void initialiseConnection() {
         try {
             int scroll_type = 1004; // defaults to ResultSet.TYPE_SCROLL_INSENSITIVE
             int concurrency_mode = 1007; // defaults to ResultSet.CONCUR_READ_ONLY
-            if (properties.containsKey(RS_SCROLL_TYPE)) scroll_type = Integer.getInteger(properties.getProperty(RS_SCROLL_TYPE));
-            if (properties.containsKey(RS_SCROLL_TYPE)) concurrency_mode = Integer.getInteger(properties.getProperty(RS_CONCURRENCY_MODE));
+            if (properties.containsKey(RS_SCROLL_TYPE)) {
+                scroll_type = Integer.getInteger(properties.getProperty(RS_SCROLL_TYPE));
+            }
```

Comment:
\[Checkstyle\] WARNING: 'if rcurly' have incorrect indentation level 12, expected level should be 6\.